mvn clean install broken as the javadoc does not build with recent JDKs (modules related)

Upgraded my JDK and routinely recompiled my git projects.

RSO does not build:

'The code being documented uses modules but the packages defined in https://docs.oracle.com/en/java/javase/13/docs/api/ are in the unnamed module.
'
The problem has been seen in other projects and is fixed by using a current version of the maven-javadoc-plugin.

We currently use `<version>2.10.4</version>` but the problem is fixed with the current
version which is <version>3.2.0</version>.

Error:

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Eclipse MicroProfile Reactive Streams Operators 1.1-SNAPSHOT:
[INFO] 
[INFO] Eclipse MicroProfile Reactive Streams Operators .... SUCCESS [  3.048 s]
[INFO] Eclipse MicroProfile Reactive Streams Operators API  FAILURE [  5.419 s]
[INFO] MicroProfile Reactive Streams Operators Core ....... SKIPPED
[INFO] Eclipse MicroProfile Reactive Streams Operators TCK  SKIPPED
[INFO] Eclipse MicroProfile Reactive Streams Operators Specification SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  8.713 s
[INFO] Finished at: 2020-06-05T11:37:51+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:jar (attach-javadocs) on project microprofile-reactive-streams-operators-api: MavenReportException: Error while generating Javadoc: 
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in https://docs.oracle.com/en/java/javase/13/docs/api/ are in the unnamed module.

$ java -version
openjdk version "14.0.1" 2020-04-14
OpenJDK Runtime Environment AdoptOpenJDK (build 14.0.1+7)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 14.0.1+7, mixed mode, sharing)

I have a PR which I have tested fixes the problem and builds lovely javadoc.

--- a/pom.xml
+++ b/pom.xml
@@ -191,7 +191,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>2.10.4</version>
+                    <version>3.2.0</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>

